docs/UserGuide.md

```diff
@@ -156,42 +244,55 @@ Format: `delete INDEX`
 
 Examples:
 * `list` followed by `delete 2` deletes the 2nd client in the F.A.R.T book.
-* `find Betsy` followed by `delete 1` deletes the 1st client in the results of the `find` command.
+* `find n/Betsy` followed by `delete 1` deletes the 1st client in the results of the `find` command.
 
 **Note:**
-1. As a `delete` command cannot be undone, the F.A.R.T Book will first prompt you to confirm if the
- contact that you want to delete is correct. If you select `No`, the operation will be aborted and the client will not be deleted.
+1. A `delete` command cannot be undone, and deleted contacts cannot be retrieved.
+1. When a `delete` command is inputted, the F.A.R.T Book will first prompt you to confirm if the contact that you want to delete is correct. If you select `No`, the operation will be aborted and the client will not be deleted. Else, select `Yes` to delete the contact.
```

Comment:
should change contact to client instead in this case

Reply:
Noted on this.